src/App.jsx

```diff
@@ -1,104 +1,27 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
```

Comment:
Ensure that the `comments.json` file exists in the `./api/` directory and contains the expected data structure.